purchase_order_product_recommendation_secondary_unit/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2019 David Vidal <david.vidal@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name":
+    "Purchase Order Product Recommendation Secondary Unit",
+    "summary":
+    "Add secondary unit to recommend products wizard",
+    "version":
+    "11.0.1.0.0",
```

Comment:
Why this weird indentation?

Reply:
Grrr, blame my vscode autopep config :disappointed: